### PR TITLE
Fix status tab not showing content

### PR DESCRIPTION
**Pull request by helfi92**
Fixes https://github.com/taskcluster/taskcluster-tools/issues/517.

### CODE REVIEW

File: src/views/AwsProvisioner/WorkerTypeStatus.jsx [hunk 1]
```diff
@@ -39,10 +36,14 @@ export default class WorkerTypeStatus extends Component {
           <code>{availabilityZone}</code>
         </td>
         <td>
-          {running * instTypeDef.capacity} ({running} instances)
+          {running
```
Comment (eliperelman) [inline src/views/AwsProvisioner/WorkerTypeStatus.jsx]: This just outputs a zero. How is this preferable to what was here before?

File: src/views/AwsProvisioner/WorkerTypeStatus.jsx [hunk 2]
```diff
@@ -56,6 +57,12 @@ export default class WorkerTypeStatus extends Component {
       ])
     ];
 
+    if (!availabilityZones.length) {
+      return (
+        <div className={emptyStatusTab}>Availability zones not available</div>
```
Comment (eliperelman) [inline src/views/AwsProvisioner/WorkerTypeStatus.jsx]: Is this messaging accurate? In what situation would there be no instances and no requests?

Comment (djmitche) [reply src/views/AwsProvisioner/WorkerTypeStatus.jsx]: When there's no work to do, and the existing instances have terminated, there will be no instances and no requests

Comment (eliperelman) [reply src/views/AwsProvisioner/WorkerTypeStatus.jsx]: OK, maybe we can have better messaging to reflect that, such as "No running availability zones" or something. This message makes me think that something went wrong.

Comment (djmitche) [reply src/views/AwsProvisioner/WorkerTypeStatus.jsx]: Perhaps "no running instances"?

File: src/views/AwsProvisioner/WorkerTypeStatus.jsx (hunk 1 above)
Comment (helfi92) [reply src/views/AwsProvisioner/WorkerTypeStatus.jsx]: Having multiple cell entries with `0 (0 instances)` made my eyes bleed.

**Pull request merged**